doc/sailfish-webview-webview.qdoc

```diff
@@ -38,32 +38,98 @@
   }
   \endcode
 
+  Advanced users may wish to make use of the
```

Comment:
It's pre-existing documentation, but this might be a good opportunity to change the `Page {` on line 29 to `WebViewPage {` to match the changes [here](https://github.com/sailfishos/sailfish-components-webview/pull/110/files#diff-408a688734bc5be5f601977c6ef0c0a895377aaafcf48f13c27f79405ec282e0R29)?

Reply:
After rebase, this is already done.